lightyear/src/transport/websocket/server.rs

```diff
@@ -87,7 +87,7 @@ impl Transport for WebSocketServerSocket {
                     let clientbound_tx_map = clientbound_tx_map.clone();
                     let serverbound_tx = serverbound_tx.clone();
                     IoTaskPool::get()
-                        .spawn(Compat::new(async move {
```

Comment:
Already in the scope of async runtime, so it is unnecessary. This whole thing is closure passed to `Compat::new` above.